webtasks/auth0_lock_webtask.js

```diff
@@ -0,0 +1,82 @@
+'use latest';
+
+const hbs = require('handlebars');
+
+/** Webtask proxy to authenticate with auth0
+ * @param TASK_URL Task to authenticate
+ */
+
+const VIEW = hbs.compile(`
+    <!DOCTYPE html>
+    <html>
+      <head>
+        <meta charset="utf8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=0" />
+
+        <title>Authenticate</title>
+
+        <script src="//cdn.auth0.com/js/lock-7.6.2.min.js"></script>
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/then-request/2.1.1/request.min.js"></script>
+      </head>
+
+      <body>
+        <script type="application/javascript">
+            window.TASK_URL     = '{{{TASK_URL}}}';
+            window.CLIENT_ID    = '{{{CLIENT_ID}}}';
+            window.AUTH0_DOMAIN = '{{{AUTH0_DOMAIN}}}';
+
+            window.onload = {{{FUNC_TO_RUN}}};
+        </script>
+      </body>
+    </html>
+`);
+
+const FUNC_TO_RUN = function() {
+    var lock = new Auth0Lock(CLIENT_ID || 'R5medkAvXYx7CC3DqUVIkWkmbs5NYv7z', AUTH0_DOMAIN || 'milomord.eu.auth0.com');
```

Comment:
Let's use `webtaskit.auth0.com` and client id `ODGT86Z8Sx0e92shNVn9N5H8JNGAh8R9`